.github/workflows/doxygen.yml

```diff
@@ -7,17 +7,19 @@ on:
 jobs:
   job:
     runs-on: ubuntu-18.04
+    container:
+      image: docker://thejohnfreeman/rippled-build-ubuntu:2944b78d22db
```

Comment:
This is fine for now, but we should probably move to hosting this container under a Ripple account.